bundles/io/org.eclipse.smarthome.io.http.auth.form/web/index.html

```diff
@@ -0,0 +1,28 @@
+<html>
+<head>
+<title>Login to Smart Home</title>
+</head>
+<body>
+
+<div id="login_form">
+    <form name="login-form" method="post" action="/login/form/index.html?process" id="f1">
```

Comment:
Better don't use an absolute, but a relative url here - `index.html?process` should just be fine.